langs/python.go

```diff
@@ -75,8 +75,7 @@ func (h *PythonLangHelper) DockerfileBuildCmds() []string {
 	r = append(r, "ADD . /function/")
 	if exists("requirements.txt") {
 		r = append(r, `
-RUN apt-get update && apt-get install --no-install-recommends -qy build-essential gcc &&\
-    pip3 install --no-cache --no-cache-dir -r requirements.txt &&\
+RUN pip3 install --target /python/  --no-cache --no-cache-dir -r requirements.txt &&\
     apt-get remove -y --purge build-essential gcc && apt-get autoremove -y &&\
```

Comment:
Think you can drop the following three lines as this doesn't install packages any more.